packages/node-resolve/README.md

```diff
@@ -47,40 +47,44 @@ Then call `rollup` either via the [CLI](https://www.rollupjs.org/guide/en/#comma
 ### `mainFields`
 
 Type: `Array[...String]`<br>
-Default: `['module', 'main']`
+Default: `['module', 'main']`<br>
+Valid values: `['browser', 'jsnext', 'module', 'main']`
 
-The fields to scan in a package.json to determine the entry point if this list contains "browser", overrides specified in "pkg.browser" will be used
+The fields to scan in a package.json to determine the entry point. Order of fields is significant, first found field would become the resolved entry point. If this list contains "browser", overrides specified in `pkg.browser` will be used.
 
 ### `module`
 
 Type: `Boolean`<br>
 Default: `true`
 
 DEPRECATED: use "mainFields" instead
-Use "module" field for ES6 module if possible
+
+Use `pkg.module` field for ES6 module if possible. This option takes precedence over both "jsnext" and "main" in the list if such are present.
 
 ### `jsnext`
 
 Type: `Boolean`<br>
 Default: `false`
 
 DEPRECATED: use "mainFields" instead
-Use "jsnext:main" if possible, legacy field pointing to ES6 module in third-party libraries, deprecated in favor of "pkg.module", see: https://github.com/rollup/rollup/wiki/pkg.module
+
+Use `pkg['jsnext:main']` if possible, legacy field pointing to ES6 module in third-party libraries, deprecated in favor of `pkg.module`, see: https://github.com/rollup/rollup/wiki/pkg.module. This option takes precedence over "main" in the list if such is present.
 
 ### `main`
 
 Type: `Boolean`<br>
 Default: `true`
 
 DEPRECATED: use "mainFields" instead
-Use "main" field or index.js, even if it's not an ES6 module (needs to be converted from CommonJS to ES6) – see https://github.com/rollup/rollup-plugin-commonjs
+
+Use `pkg.main` field or index.js, even if it's not an ES6 module (needs to be converted from CommonJS to ES6), see https://github.com/rollup/rollup-plugin-commonjs.
```

Comment:
```suggestion
```

Let's remove the description of this option, it's Type and Default descriptor as well, but leave the DEPRECATED notice, since it's deprecated.

Reply:
f4da5de